docs/src/main/sphinx/admin/properties-write-partitioning.rst

```diff
@@ -12,15 +12,18 @@ minimum number of partitions, set in ``preferred-write-partitioning-min-number-o
 are written, each partition is written by a separate writer. As a result, for some connectors such as the
 Hive connector, only a single new file is written per partition, instead of
 multiple files. Partition writer assignments are distributed across worker
-nodes for parallel processing.
+nodes for parallel processing. This can be specified on a per-query basis using
```

Comment:
``use-preferred-write-partitioning`` can be specified on a per...
Similar for the other one

Reply:
done